examples/org.eclipse.emfcloud.modelserver.example/pom.xml

```diff
@@ -64,12 +64,12 @@
          <version>${project.version}</version>
          <scope>provided</scope>
       </dependency>
-      <dependency>
+      <!--<dependency>
```

Comment:
Why do we remove this lib dependency?

Reply:
it's not needed as it is pulled in by the other dependencies. We get rid of many warnings by removing it.